source/MaterialXRuntime/RtApi.h

```diff
@@ -75,17 +76,19 @@ class RtApi
     RtPrimIterator getMasterPrims(RtObjectPredicate predicate = nullptr);
 
     /// Register a typed prim schema.
-    template<class T>
+    template<class T, class ConnectableApi = RtConnectableApi>
```

Comment:
Set a connectable API to use (optional) when registering the prim schema. The default implementation is set if none is given.